### PR TITLE
dbm instead of gdbm (Python 2)

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

**Comment by mkurek** [comment]
:ok:

File: contrib/dhcp_agent/dhcp_agent.py [hunk 1]
```diff
@@ -16,13 +16,13 @@
 
 IS_PY3 = sys.version_info[0] == 3
 if IS_PY3:
-    from dbm import gnu as gdbm
+    from dbm import gnu as cache_db
```
Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py]: \[Pylint\] INFO: Import "from dbm import gnu as cache\_db" should be placed at the top of the module \[wrong\-import\-position\]

File: contrib/dhcp_agent/dhcp_agent.py [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
     from urllib.request import urlopen, Request
     from urllib.parse import urlencode
     from urllib.error import HTTPError
     string_types = (str,)
 else:
-    import gdbm
+    import dbm as cache_db
```
Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py]: \[Pylint\] INFO: Import "import dbm as cache\_db" should be placed at the top of the module \[wrong\-import\-position\]

Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py]: \[Pylint\] INFO: Imports from package dbm are not grouped \[ungrouped\-imports\]

File: contrib/dhcp_agent/dhcp_agent.py (hunk 1 above)
Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py]: \[Pylint\] INFO: Import "from dbm import gnu as cache\_db" should be placed at the top of the module \[wrong\-import\-position\]

File: contrib/dhcp_agent/dhcp_agent.py (hunk 2 above)
Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py]: \[Pylint\] INFO: Import "import dbm as cache\_db" should be placed at the top of the module \[wrong\-import\-position\]

Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py]: \[Pylint\] INFO: Imports from package dbm are not grouped \[ungrouped\-imports\]

File: contrib/dhcp_agent/dhcp_agent.py (hunk 1 above)
Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py:19]: [Pylint] INFO: Import "from dbm import gnu as cache_db" should be placed at the top of the module [wrong-import-position]

File: contrib/dhcp_agent/dhcp_agent.py (hunk 2 above)
Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py:25]: [Pylint] INFO: Import "import dbm as cache_db" should be placed at the top of the module [wrong-import-position]

Comment (sputnikci) [inline contrib/dhcp_agent/dhcp_agent.py:25]: [Pylint] INFO: Imports from package dbm are not grouped [ungrouped-imports]

**Pull request merged**